unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDView.cs

```diff
@@ -452,7 +453,8 @@ private void SortEntriesImmediate()
         {
             if (this.entries.Count <= 0) return;
 
-            var entries = this.entries.Values.OrderBy(x => x.Model.timestamp).ToList();
+            var entries = this.entries.Values.ToList();
+            entries.Sort((a, b) => SortingStrategy.Invoke(a.Model, b.Model));
```

Comment:
Won't this cause allocations?

Reply:
I dont think there is any new allocations that we already had. We should try to avoid `ToList()` but its needed for the `SiblingIndex` later. Perhaps there is a way to avoid it. Im not super worried because the sorting will be triggered one time per frame at MOST (which is not likely to happen), and we have a max of 30 entries.

Reply:
Just uploaded these changes, i like it

Reply:
Cool. Doesn't the lambda closure creates an allocation? I mean the `() => {}` syntax

Reply:
Not the lambda. But `OrderBy` creates allocations